cmd/context/delete.go

```diff
@@ -28,40 +30,59 @@ import (
 func DeleteCMD() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "delete",
-		Args:  utils.ExactArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
-		Short: "Delete a context",
+		Args:  utils.MinimumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
+		Short: "Delete one or more contexts",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			args[0] = okteto.AddSchema(args[0])
-			args[0] = strings.TrimSuffix(args[0], "/")
-			return Delete(args[0])
+			for idx, arg := range args {
+				args[idx] = okteto.AddSchema(arg)
+				args[idx] = strings.TrimSuffix(arg, "/")
+			}
+			errs := Delete(args)
+			var errLen int
+			var totalContextsDeleted int
+			if errs != nil {
+				if merr, ok := errs.(*multierror.Error); ok {
+					errLen = len(merr.Errors)
+				}
+			}
+			totalContextsDeleted = len(args) - errLen
+			if len(args) == errLen {
+				analytics.TrackContextDelete(totalContextsDeleted, false)
+			}
+			analytics.TrackContextDelete(totalContextsDeleted, true)
+			return errs
 		},
 	}
 	return cmd
 }
 
-func Delete(okCtx string) error {
+func Delete(okCtxs []string) error {
```

Comment:
In order to return a proper output following the criteria and the hint we want to display, let's use the following `Delete` func:
````go
func Delete(okCtxs []string) error {
	ctxStore := okteto.ContextStore()
	var errs error
	validOptions := make([]string, 0)
	for _, okCtx := range okCtxs {
		if okCtx == ctxStore.CurrentContext {
			ctxStore.CurrentContext = ""
		}

		if _, ok := ctxStore.Contexts[okCtx]; ok {
			delete(ctxStore.Contexts, okCtx)
			if err := okteto.NewContextConfigWriter().Write(); err != nil {
				return err
			}
			oktetoLog.Success("'%s' deleted successfully", okCtx)
		} else {
			for k, v := range ctxStore.Contexts {
				if v.IsOkteto {
					validOptions = append(validOptions, k)
				}
			}
			errs = multierror.Append(errs, fmt.Errorf("'%s' context doesn't exist.", okCtx))
		}
	}

	if errs != nil {
		if err, ok := errs.(*multierror.Error); ok {
			err.ErrorFormat = func(es []error) string {
				points := make([]string, len(es))
				for i, err := range es {
					points[i] = fmt.Sprintf("\t- %s", err)
				}

				hint := fmt.Sprintf("Valid options are: [%s]. To delete a Kubernetes context run 'kubectl config delete-context <k8s-context-name>'", strings.Join(validOptions, ", "))
				return fmt.Sprintf("Following contexts couldn't be deleted:\n%s\n%s\n\n", strings.Join(points, "\n"), hint)
			}
		}
	}

	return errs
}
````
output will be:
<img width="1477" alt="Captura de Pantalla 2023-09-06 a las 16 51 32" src="https://github.com/okteto/okteto/assets/22500940/1e8684ee-b739-42c3-8b18-03a6e2604607">